pyspcwebgw/__init__.py

```diff
@@ -44,10 +44,12 @@ def zones(self):
 
     def start(self):
         """Connect websocket to SPC Web Gateway."""
-        self._websocket = AIOWSClient(loop=self._loop,
-                                      session=self._session,
-                                      url=self._ws_url,
-                                      async_callback=self._async_ws_handler)
+        self._websocket = AIOWSClient(
+            loop=self._loop,
+            session=self._session,
+            url=self._ws_url,
+            async_callback=self._async_ws_handler,
```

Comment:
extra comma sign

Reply:
yes, that is how `black` formats things.